packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -334,8 +334,9 @@ export class CucumberJSAllureFormatter extends Formatter {
       return;
     }
 
-    const { fileName = "attachment", body, mediaType } = data;
-    const attachmentFilename = this.allureRuntime.writeAttachment(body, mediaType);
+    const { fileName = "attachment", body, mediaType, contentEncoding } = data;
```

Comment:
CucumberJs passes contentEncoding as either "IDENTITY" or "BASE64" here.  Because I wasn't sure how other libraries like Mocha, Jest, or Playwright responded, I let the default option as 'utf-8'